test/bcccp/carpark/CarparkTest.java

```diff
@@ -23,7 +27,7 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-@RunWith(MockitoJUnitRunner.class)
```

Comment:
you have made some minor changes to the carpark test class. was mockito not working for you?

Reply:
No changes made to carpark - at least not intentionally. That is a worry. I may need to re-fetch and pull carpark to my local master and then re-merge with my local branch, etc. Also, I have noticed that CarparkTest includes some strange corruptions in the code, see lines 65, 68 and 72 - these look like git commands/output have somehow been merged into the code - not sure how that could have happened.